ui/apps/platform/cypress/integration/configmanagement/ciscontrols.test.js

```diff
@@ -6,63 +6,73 @@ import {
     hasTabsFor,
     pageEntityCountMatchesTableRows,
     sidePanelEntityCountMatchesTableRows,
+    interactAndWaitForConfigurationManagementScan,
+    visitConfigurationManagementDashboard,
+    visitConfigurationManagementEntities,
 } from '../../helpers/configWorkflowUtils';
 import {
-    url,
     selectors as configManagementSelectors,
     controlStatus,
 } from '../../constants/ConfigManagementPage';
 import withAuth from '../../helpers/basicAuth';
-import { triggerScan } from '../../helpers/compliance';
+
+// const entitiesKey = 'controls'; // omit to minimize changed lines
```

Comment:
Not sure it's worth delaying this change by commenting this pattern in each of the files.

Reply:
I will open a follow up use constants and to split `it` with pair of `context` into pair of `it` tests.